windows/RNSVG/BrushView.cpp

```diff
@@ -10,14 +10,13 @@ void BrushView::SaveDefinition() {
   }
 }
 
-void BrushView::SetBounds(Windows::Foundation::Rect const &rect) {
-  m_bounds = rect;
+void BrushView::SetBounds(Rect const &rect) {
+  m_bounds = {rect.X, rect.Y, rect.Width + rect.X, rect.Height + rect.Y};
   UpdateBounds();
 }
 
 void BrushView::Unload() {
   if (m_brush) {
-    m_brush.Close();
```

Comment:
Just curious, why did we close the brush before if we set it to nullptr in the next line?

Reply:
m_brush used to be an ICanvasBrush, now it's something else.
The ICanvasBrush interface implements IDisposable (WinRT IClosable), which requires calling Close before releasing the last reference:

- https://microsoft.github.io/Win2D/WinUI3/html/T_Microsoft_Graphics_Canvas_Brushes_ICanvasBrush.htm

- https://learn.microsoft.com/en-us/uwp/api/windows.foundation.iclosable?view=winrt-22621